### PR TITLE
Progress Report - March 2023

### PR DESCRIPTION
**Pull request by goldenx86**

### CODE REVIEW

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+## Other code changes
+
+Fire Emblem fans have been experiencing freezes in game, and after some investigation, Morph found out that the culprit was the bounded threadsafe queue, introduced with a pull request from [behunin](https://github.com/behunin).
+
+The queue’s job is to pass events from an event producer (like the CPU cores) to an event consumer (like the GPU thread).
+The previous implementation which behunin replaced kept allocating and freeing memory, causing periodic latency spikes.
+
+The way to solve this issue is to turn the queue into a {{< gh-hovercard "9778" "bounded queue," >}} meaning no memory allocation is needed as there is a fixed amount of resources available.
+And that’s what behunin did, but the new queue implementation seemed to be buggy. 
+Morph added a {{< gh-hovercard "9971" "simplified implementation," >}}, fixing the new gameplay freezes affecting `Fire Emblem: Three Houses` and `Fire Emblem Engage`.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:309]:
```suggestion
Morph added a {{< gh-hovercard "9971" "simplified implementation," >}} fixing the new gameplay freezes affecting `Fire Emblem: Three Houses` and `Fire Emblem Engage`.
```

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+
+[Your writer](https://github.com/goldenx86) re-enabled LTO for GCC builds, following the changes made for Windows last time, but only applying the {{< gh-hovercard " optimizations" >}} to the core and video_core subprojects, giving Linux users a nice free performance boost.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:311]:
```suggestion
[Your writer](https://github.com/goldenx86) re-enabled LTO for GCC builds, following the changes made for Windows last time, but only applying the {{< gh-hovercard "10014" "optimizations to the core and video_core subprojects," >}} providing Linux users a nice, free performance boost.
```

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+
+## Hardware section
+
+### AMD 23.3.2 and newer drivers
+
+One bad and two good news.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:317]:
```suggestion
One bad and two good pieces of news.
```

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+
+On the positive side, AMD has finally added support for the missing dynamic state extensions in their latest Windows driver release, which improves the shader build times and reduces the stuttering on Radeon GPUs, making them perform on par with NVIDIA.
+
+However, on the negative side, this driver update also introduces a serious regression:
+
+{{< imgs
+	"./amd.png| This can affect your entire desktop (Metroid Prime Remastered)"
+  >}}
+
+Some games, such as `Metroid Prime Remastered` and `Pokémon Legends: Arceus` trigger what seems to be an out-of-bounds crash that causes desktop corruption and a driver crash.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:327]:
```suggestion
Some games, such as `Metroid Prime Remastered` and `Pokémon Legends: Arceus`, trigger what seems to be an out-of-bounds crash that causes desktop corruption and a driver crash.
```

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+
+We reported this issue to AMD and they promptly confirmed it, reproduced it, and informed us that they are working on a fix.
+
+Now *that’s* how you do customer support.
+
+Speaking of which…
+
+### Doing Intel’s job
+
+While testing a crucial rendering fix for `The Legend of Zelda: Breath of the Wild` that we will discuss next month, Morph did some feature testing to investigate what makes Intel Windows drivers run like the Challenger project.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:338]:
```suggestion
While testing a crucial rendering fix for `The Legend of Zelda: Breath of the Wild` that we will discuss next month, Morph did some feature testing to investigate what makes Intel Windows drivers run so horribly.
```
Removed reference to shuttle explosion that involves death

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 6]
```diff
@@ -0,0 +1,383 @@
...
+	"./gaia.jpg| Flute intensifies"
+  >}}
+
+Project Gaia is still in development.
+
+Oh and Project Lime is in release candidate status, under intense internal testing.
+
+That’s all folks! Thank you for sticking until the end.
+
+Thanks to Bing Chat for the terrible jokes.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:378]:
```suggestion
{{< imgs
	"./gaia.jpg| Flute intensifies"
  >}}

Regarding Project Gaia:
Project Gaia is still in development.

Oh, and Project Lime is in release candidate status, under intense internal testing.

That’s all folks! Thank you for sticking around until the end of this progress report.

Thanks to Bing Chat for the terrible jokes.
```

**Review by Schplee — CHANGES_REQUESTED** [review]

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 7]
```diff
@@ -0,0 +1,383 @@
...
+    "Left: old 1ms precision; Right: current 0.5ms precision. The result comes from your writer’s laptop which runs a Single Language Windows installation, so don’t mind the Spanish. Note that there is a difference in CPU use between the two results"
+    "./usebug.png"
+    "./usefix.png"
+    >}}
+
+If you can perform more tasks in the same power envelope, you can improve performance without sacrificing battery life.
+Alternatively, a more common scenario for an emulator, if your ceiling is a fixed framerate, you can reduce temperatures.
+
+{{< single-title-imgs
+    "This is why the lap in laptop is a lie, mind the package temperature (Xenoblade Chronicles 3)"
```
[the remark this answers is not in the document]
Comment (goldenx86) [reply site/content/entry/yuzu-progress-report-mar-2023/index.md:47]: This breaks the image tag.

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 8]
```diff
@@ -0,0 +1,382 @@
...
+Most likely Nintendo intends to use it for their internal emulators, maybe for "backwards compatibility."
+Regardless of any of that, we have a lot of work ahead of us to add support for this extension.
+
+But enough about that. Let’s talk about some exciting projects in the works.
+{{< imgs
+	"./gaia.jpg| Flute intensifies"
+  >}}
+
+Regarding Project Gaia:
+Project Gaia is still in development.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:371]:
```suggestion
it is still in development.
```

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 9]
```diff
@@ -0,0 +1,382 @@
...
+
+Changing topics to Amiibo support, a lot has happened this month.
+
+First of all, german77 managed to {{< gh-hovercard "9953" "write" >}} the correct [CRC](https://en.wikipedia.org/wiki/Cyclic_redundancy_check), making Amiibo data written by yuzu compatible with the Switch; no more invalid info.
+
+Next, if you tried to connect a controller while scanning an Amiibo, it would not be recognized. By {{< gh-hovercard "9981" "accounting" >}} for this and initializing or finalizing the controller, he fixed the issue.
+
+And last but not least, to wrap up the Amiibo saga, Herman the German Sherman {{< gh-hovercard "9995" "added support" >}} for plain Amiibo files.
+Some Amiibos, like the ones themed after `Super Smash Bros. Ultimate`, don’t encrypt their data, so they don’t need keys to be dumped. 
+Now yuzu can read non-encrypted game data from your Amiibos correctly!
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:287]:
```suggestion
Some Amiibos, like the ones themed after `Super Smash Bros. Ultimate`, can be edited without the need of keys. Now yuzu can read non-encrypted game data from your Amiibos correctly!
```

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 10]
```diff
@@ -0,0 +1,382 @@
++++
+date = "2023-04-05T12:00:00-03:00"
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:2]:
```suggestion
date = "2023-04-10T12:00:00-03:00"
```

File: site/content/entry/yuzu-progress-report-mar-2023/index.md [hunk 11]
```diff
@@ -0,0 +1,382 @@
...
+title = "Progress Report March 2023"
+author = "GoldenX86"
+forum = 0
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-mar-2023/index.md:5]:
```suggestion
forum = 756599
```

**Review by Schplee — CHANGES_REQUESTED** [review]

**Pull request merged**